### PR TITLE
Add Find Seat 

### PR DESCRIPTION
**Pull request by rwema3**
1. [ ] Project URL:
2. [ ] Update contents.json instead of README 
3. [ ] One project per pull request
4. [ ] Screenshot included 
5. [ ] Avoid iOS or open-source in description as it is assumed
6. [ ] Use this commit title format if applicable: Add app-name by @github-username
7. [ ] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -30207,6 +30226,22 @@
       "suggested_by": "@dkhamsing",
       "stars": 230
     },
+    {
+      "title": "Space Empires",
```
Comment (dkhamsing) [inline contents.json:30230]: Can you remove the space empires entry ?

**Review by dkhamsing — CHANGES_REQUESTED** [review]

File: contents.json (hunk 1 above)
Comment (rwema3) [reply contents.json:30230]: OHH still there!!

Comment (rwema3) [reply contents.json:30230]: done

Comment (dkhamsing) [reply contents.json:30230]: Done where? What is the commit

**Comment by rwema3** [comment]
donee!!

**Comment by dkhamsing** [comment]
Thx!

**Pull request merged**